Mathlib/Algebra/Lie/Semisimple/Defs.lean

```diff
@@ -29,6 +29,14 @@ variable [CommRing R] [LieRing L] [AddCommGroup M] [Module R M] [LieRingModule L
 abbrev LieModule.IsIrreducible : Prop :=
   IsSimpleOrder (LieSubmodule R L M)
 
+variable {R L M} in
+lemma LieModule.IsIrreducible.mk [Nontrivial M] (h : ∀ N : LieSubmodule R L M, N ≠ ⊥ → N = ⊤) :
```

Comment:
Should this be a constructor for `IsSimpleOrder` instead?

Reply:
(or maybe just removed, since this is the existing constructor modulo `or_iff_not_imp_left`)

bors d+

Thanks!